perch/model.py

```diff
@@ -373,6 +463,33 @@ def id(self):
     def id(self, value):
         self._resource['id'] = value
 
+    @classmethod
+    @coroutine
+    def create(cls, user, **kwargs):
+        """If parent resource is not an editable state, should not be able to create."""
+        parent_id = kwargs.get(cls.parent_resource.resource_type + '_id')
+        parent = yield cls.parent_resource.get(parent_id)
+
+        if parent.state not in cls.editable_states:
```

Comment:
Couldn't `parent` have different `editable_states`?

I think we should add an `editable` property so that it's possible to do 

```python
if not parent.editable:
    ...
```